rest/data/interface.go

```diff
@@ -29,6 +29,20 @@ import (
 	"github.com/mongodb/amboy"
 )
 
+// FindTestsByTaskIdOpts contains filtering, sorting and pagination options for TestResults.
+type FindTestsByTaskIdOpts struct {
+	Execution int    // Execution matches exact.
```

Comment:
I think you can remove the individual field level documentation—they are pretty self-explanatory. If you keep anything, move it to the line above the field.